File: week6/echon.py
#!/usr/bin/python

import sys

if len(sys.argv) != 3:
    print >>sys.stderr, "Usage: %s" % sys.argv[0]
    sys.exit(1)

for x in range(int(sys.argv[1])):
    print "%s" % sys.argv[2]

File: week5/echon.pl
#!/usr/bin/perl -w

if (@ARGV == 2) {
	for ($i = 0; $i < $ARGV[0]; $i++) {
		print "$ARGV[1]\n";
	}
} else {
	print "Usage: ./echon.pl <number of lines> <string>\n";
}